src/errors/index.ts

```diff
@@ -1,10 +1,5 @@
-export enum ErrorReason {
-  Unkown = 'Unknow Error',
-  ReponseFailed = 'Reponse Failed',
-}
-
-export type TimeTacApiError = {
-  reason: ErrorReason;
-  response?: unknown;
-  _plainError?: string;
+export type ErrorFormat = {
+  statusCode: number | undefined;
```

Comment:
```suggestion
  statusCode?: number;
```
Or does this not work?